src/ecommerce/package.json

```diff
@@ -22,7 +22,7 @@
   "devDependencies": {
     "eslint": "7.29.0",
     "eslint-config-next": "11.0.0",
-    "prettier": "^2.6.2"
+    "prettier": "^2.6.2",
+    "standard": "^17.0.0"
```

Comment:
What is this for?

Reply:
@mdubus standard is a package that allows to check for issues related to inconsistency in semi colons, quotes etc. in the entire code. 
https://www.npmjs.com/package/standard

Reply:
Alright, so I will just remove the standard package and ensure to follow the eslint package only for linting the files.